### DISCUSSION TITLE
Documentations for the `!inner` keyword?

### DISCUSSION THREAD

**Discussion by laubonghaudoi** [supabase/supabase#13944] [Questions] [answered]
Original question from https://github.com/orgs/supabase/discussions/12900 . I couldn't find any documentation about this `!inner` trick. What api is this and is there any documentation for that?

**Comment by GaryAustin1** [comment, marked as answer]
From the javascript client docs...
![image](https://user-images.githubusercontent.com/54564956/234154272-eaad99a2-4c92-4fb6-9755-4a65dd47d30d.png)

And from PostgREST docs in this section.
https://postgrest.org/en/stable/api.html#embedding-with-top-level-filtering

**Reply by laubonghaudoi** [reply]
Thanks! I was looking for an example but couldn't find it in the supabase docs. I think the supabase-js doc should link to the PostgREST doc so that new users are aware that the apis are actually available in PostgREST

**Reply by GaryAustin1** [reply]
It is in a few places.  Example below goes to the PostgREST docs.
![image](https://user-images.githubusercontent.com/54564956/234309196-8c965139-606b-45ca-b7f7-b31ac848dc24.png)
But if you have specific place you want to see it, and you want someone to possibly act on it, then doc issues are generated in supabase/supabase issues.

**Reply by laubonghaudoi** [reply]
Interesting, because when I search the `inner` keyword, it got no results:
<img width="769" alt="image" src="https://user-images.githubusercontent.com/11172180/234351910-1734aedb-0ab1-4d8a-898d-24489fe08d88.png">
And searching for "filtering foreign tables" got unhelpful results too:
<img width="764" alt="image" src="https://user-images.githubusercontent.com/11172180/234352433-59ab217c-73c8-45a7-9e5f-7b96d0e5d81f.png">
I guess the search function of the doc site is just lame. As new users we have to read the whole doc line by line or get very familiar with each doc.
